### ISSUE TITLE
Time stamp showing "Now" when it wasn't now

### ISSUE THREAD

**Issue by tcbren** [klinker-apps/messenger-issues#716]
**Device with issue:** Pixel 2 XL on Android 8.1
**Primary device:** Pixel 2 XL on Android 8.1
**App version:** 4.0.0.2174
**Steps to reproduce:**
Don't know what reproduces it, but it happens frequently.

**Additional information:**

Often the most recent text, or several most recent, when tapped will show "Now" as when it was received, even if it was received minutes, or hours previously. Only way to see actual receipt time is to long press, and click info button. 

Not just this device or version. Has happened for many versions, and on my old phone, Nexus 6P, as well.

**Comment by klinker24** [comment]
You could scroll the message off the screen, or close and re-open the conversation to update the timestamp.

**Comment by Skullmonkey** [comment]
When I read a message and then lock my phone without closing the conversation, I can unlock my phone hours later and it still show "Now". Or if I don't notice that I have a text and don't read it until hours later, I can open my phone and it say "Now" even though it was hours ago. Granted, I have gotten used to scrolling the conversation and then scrolling back to see when it was actually sent, but it would be really nice to be able to not have to do that.

My suggestion would be to have the time be relative up to an hour. After an hour, it shows absolute time. 
Ex:
1 minute ago
2 minutes ago
3 minutes ago
...
58 minutes ago
59 minutes ago
9:18 AM

Some reading material for relative time: https://stackoverflow.com/questions/19409035/custom-format-for-relative-time-span

TL;DR:
It would be nice to at the very least have it go to absolute time after 5 minutes or so. But if you are going to change it anyways, it would be a QoL change to have relative time.

**Comment by klinker24** [comment]
That isn't what this topic is discussing, nor do I plan on changing how the relative timestamps work, sorry. They show "now", for two minutes, then they go to the timestamp.

You can tap any message to show/hide the timestamp. On the latest version of the app, this will update the text of the timestamp. No other changes will come to this functionality, sorry.
